SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -1040,6 +1040,15 @@ SCDocHTMLRenderer {
 
 		this.renderHeader(stream, doc, body);
 		this.renderChildren(stream, body);
+
+		if(doc.isClassDoc) {
+			var moreExamples = "https://sccode.org/search?q=" ++ doc.klass;
```

Comment:
You should rather search via

```suggestion
			var moreExamples = "https://sccode.org/tag/class/%".format(doc.klass);
```

IMO text which is really similar across 1000+ of pages should be solved in a more generic manner - maybe a button or a plain link to sccode on the top of the class page?